### PR TITLE
egl: set context version when egl < 1.5

### LINKED ISSUE #1647: Unable to Force GLES2 Context (Still picks up the wrong GLES version)
My device supports GLES2, but glutin instead uses GLES1 (OpenGL ES-CM 1.1), even when I force it via context:
```rust
let context_attributes = ContextAttributesBuilder::new()
            .with_context_api(ContextApi::Gles(Some(glutin::context::Version::new(2, 0))))
            .build(Some(raw_window_handle));
```

I also try to force the API level for the config:

```rs
let builder = ConfigTemplateBuilder::new()
            .with_api(glutin::config::Api::GLES2)
            .with_alpha_size(8)
            .compatible_with_native_window(raw_window_handle)
            .with_surface_type(ConfigSurfaceTypes::WINDOW);
```

Printing config.api() gives the following output, also indicating the existence of GLES2 support:
```
I/RustStdoutStderr( 3377): Config supports API: Api(GLES1 | GLES2)
```

The android device has an Ardeno 200 GPU, which has GLES2 support.

It's worth noting that I get an error about `EGL_BAD_DISPLAY` initially before anything:

```
validate_display:254 error 3008 (EGL_BAD_DISPLAY)
```

I have no idea what's causing it anymore. Any ideas are appreciated.

### PR DESCRIPTION
**Pull request by kchibisov**
Signed-off-by: p0ryae <theporyafm12@gmail.com>
Fixes #1647.

--

@p0ryae does it fix your issue?

### CODE REVIEW

File: glutin/src/api/egl/context.rs [hunk 1]
```diff
@@ -117,6 +117,12 @@ impl Display {
                 attrs.push(egl::CONTEXT_FLAGS_KHR as EGLint);
                 attrs.push(flags as EGLint);
             }
+        } else if self.inner.version >= Version::new(1, 3) {
+            // EGL 1.3 uses that to indicate client version instead of major/minor.
+            if let Some(version) = version {
+                attrs.push(egl::CONTEXT_CLIENT_VERSION as EGLint);
```
Comment (MarijnS95) [inline glutin/src/api/egl/context.rs:123]: Note that this constant is identical in value to `CONTEXT_MAJOR_VERSION` (maybe comment?).

**Review by MarijnS95 — APPROVED** [review]

**Comment by MarijnS95** [comment]
The commit signoff looks a bit weird, since you (@kchibisov) are both the author and committer.

Perhaps this should be a `Reported-by:`, `Suggested-by:`, `Tested-by:`?

**Comment by kchibisov** [comment]
>Perhaps this should be a Reported-by:, Suggested-by:, Tested-by:?

https://www.kernel.org/doc/html/latest/process/submitting-patches.html#when-to-use-acked-by-cc-and-co-developed-by

**Comment by kchibisov** [comment]
I guess `C-d-b` is better.

**Comment by MarijnS95** [comment]
Oh yeah I forgot C-d-b, that's best!

**Comment by p0ryae** [comment]
> @p0ryae does it fix your issue?

Yes. Tested the [egl-1.3-set-version](https://github.com/kchibisov/glutin/tree/egl-1.3-set-version) branch and my tests pass.

**Pull request merged**